js_modules/dagster-ui/packages/ui-core/src/assets/AutoMaterializePolicyPage/PolicyEvaluationTable.tsx

```diff
@@ -155,6 +171,56 @@ const NewPolicyEvaluationTable = ({
             startTimestamp = evaluation.startTimestamp;
           }
 
+          const assetDisplayName = assetKey ? displayNameForAssetKey(assetKey) : '';
+          const lastEvaluationForAssetKey =
+            assetKey &&
+            lastEvaluationsByAssetKey &&
+            tokenForAssetKey(assetKey) in lastEvaluationsByAssetKey
+              ? lastEvaluationsByAssetKey[tokenForAssetKey(assetKey)]
+              : null;
+          const evaluationNodeRef =
+            assetKey &&
+            assetKeyPath &&
+            pushHistory &&
+            lastEvaluationForAssetKey &&
+            tokenForAssetKey(assetKey) !== tokenForAssetKey({path: assetKeyPath}) ? (
```

Comment:
This ternary is a bit hard to read. Could you just return null early in the false case?